common/src/main/java/dev/lavalink/youtube/clients/Web.java

```diff
@@ -207,17 +207,35 @@ protected JsonBrowser extractPlaylistVideoList(@NotNull JsonBrowser json) {
     protected String extractPlaylistContinuationToken(@NotNull JsonBrowser videoList) {
         // WEB continuations seem to be slightly inconsistent.
         JsonBrowser contents = videoList.get("contents");
-
         if (!contents.isNull()) {
             videoList = contents;
         }
-
         return videoList.values()
-            .stream()
-            .filter(item -> !item.get("continuationItemRenderer").isNull())
-            .findFirst()
-            .map(item -> item.get("continuationItemRenderer").get("continuationEndpoint").get("continuationCommand").get("token").text())
-            .orElse(null);
+                .stream()
+                .filter(item -> !item.get("continuationItemRenderer").isNull())
+                .findFirst()
+                .map(item -> {
+                    JsonBrowser continuationRenderer = item.get("continuationItemRenderer");
+                    JsonBrowser continuationEndpoint = continuationRenderer.get("continuationEndpoint");
```

Comment:
This could be simplified to just `JsonBrowser continuationEndpoint = item.get("continuationItemRenderer").get("continuationEndpoint");`